moonwalking/main.py

```diff
@@ -281,3 +279,20 @@ class Lendingblock(EthereumGeneric):
             raise GeneralError("not enough eth in buffer wallet")
 
         return addr, priv
+
+    async def build_tx(self, priv: str, addrs: List[Tuple[str, D]]):
```

Comment:
Does this method ever get called now that we're fixing the base classes method in `EthereumGeneric`?